OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -406,20 +420,28 @@ protected override MovePart OnComplete(Actor self, Mobile mobile, Move parent)
 					mobile.Facing,
 					moveFraction - MoveFractionTotal);
 
+				// Raises an "event" telling subscribed units that this one is entering
+				// TODO: Maybe replace this by a real event in the future?
 				mobile.EnteringCell(self);
+
+				// Set the unit as if it arrived to its destination
 				mobile.SetLocation(mobile.ToCell, mobile.ToSubCell, mobile.ToCell, mobile.ToSubCell);
 				return ret2;
 			}
 		}
 
-		class MoveSecondHalf : MovePart
+		/// <summary>
+		/// This class does the second part of a movement. Basically, it notifies
+		/// any possible crush in the cell once movement is terminated.
```

Comment:
I suggest rewording this a bit.  The current wording seems to imply that this *only* handles that notification, which isn't true.